### ISSUE TITLE
Telegram notification needs parsing

### ISSUE THREAD

**Issue by didyouexpectthat** [tidusjar/Ombi#1686]
#### Ombi build Version:

V 3.0.2310

#### Update Branch:

DotNetcore

#### Media Server:

Plex

#### Operating System:

Debian 8.9

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

```
2017-11-05 16:48:14.410 -08:00 [Error] StatusCode: NotFound, Reason: Not Found
```

#### Problem Description:

Telegram notification text needs parsing before posting

#### Reproduction Steps:

Telegram's API does not like spaces and capital H. You'll need to replace spaces with `%20`
Ombi sent a notification to Telegram but the message had a capital H with a space before it. This is interpreted by Telegram as something else. 
Or you can apparently use some kind of urlencoding. 
https://stackoverflow.com/questions/37923586/h-letter-is-not-sending-to-telegram-bot

I had the same issue with my bash script. I was posting to Telegram's API but it was failing for this one rule for some reason. I now replace spaces with %20 and it works great on my app, but I think you'll need to implement this as well.

**Comment by tidusjar** [comment]
Are you sure that's the issue? Because I'm sending the text as part of the http requests body and not as a query string.

**Comment by didyouexpectthat** [comment]
I am not 100%... but I expected to get a Telegram notification with requested user's alias once the movie became available. Ombi left the show pending for a long time (including some other errors) so I manually marked the movie as available. Ombi sent her an email via smtp, but I never got the corresponding Telegram notification. The user's alias last name is " Hansen"

I will try later to get another notification. This was just the only thing I could think of as to why I did not receive a notification. Let me try it out some more. 

![image](https://user-images.githubusercontent.com/1329077/32448887-ffca2518-c2c4-11e7-917e-3b0e466666aa.png)
